demisto_sdk/commands/validate/validators/PA_validators/PA114_pack_metadata_version_should_be_raised.py

```diff
@@ -0,0 +1,184 @@
+    Mapper,
+    ModelingRule,
+    ParsingRule,
+    Playbook,
+    PreProcessRule,
+    Report,
+    Script,
+    Trigger,
+    Widget,
+    Wizard,
+    XDRCTemplate,
+    XSIAMDashboard,
+    XSIAMReport,
+)
+from demisto_sdk.commands.content_graph.objects.pack import Pack
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[
+    Pack,
+    Integration,
+    Script,
+    Playbook,
+    AssetsModelingRule,
+    BasePlaybook,
+    BaseScript,
+    CaseField,
+    CaseLayout,
+    CaseLayoutRule,
+    Classifier,
+    CorrelationRule,
+    Dashboard,
+    GenericDefinition,
+    GenericField,
+    GenericModule,
+    GenericType,
+    IncidentField,
+    IncidentType,
+    IndicatorField,
+    IndicatorType,
+    Job,
+    Layout,
+    LayoutRule,
+    Mapper,
+    ModelingRule,
+    Pack,
+    ParsingRule,
+    Playbook,
+    PreProcessRule,
+    Report,
+    Script,
+    Trigger,
+    Widget,
+    Wizard,
+    XDRCTemplate,
+    XSIAMDashboard,
+    XSIAMReport,
+    List,
+]
+
+
+class PackMetadataVersionShouldBeRaisedValidator(BaseValidator[ContentTypes]):
+    error_code = "PA114"
+    description = "Ensure that the pack metadata version is raised on relevant changes."
+    rationale = (
+        "When updating a pack, its version needs to be raised to maintain traceability."
+    )
+    error_message = (
+        "The pack version (currently: {old_version}) needs to be raised - "
+        "make sure you are merged from master and "
+        'update the "currentVersion" field in the '
+        "pack_metadata.json or in case release notes are required run:\n"
+        "`demisto-sdk update-release-notes -g` - for automatically generation of release notes and version\n"
+        "`demisto-sdk update-release-notes -i Packs/{pack} -u "
+        "(major|minor|revision|documentation)` for a specific pack and version."
+    )
+
+    @staticmethod
+    def should_bump_is_metadata(content_item: ContentTypes):
+        """Return if the version should be bumped as a result of the input content item.
+        Args:
+            content_item (ContentTypes): The content item for which to decide.
+
+        Return:
+            ((Boolean) True if should bump due to change False otherwise,
+             (Boolean) True if the content item is the pack_metadata False otherwise)
+        """
+        content_item_type = find_type(str(content_item.path))  # type: ignore[union-attr]
+        is_metadata = False
+
+        if (
+            not content_item_type
+            and content_item.content_type.value == Pack.content_type.value  # type: ignore[union-attr]
+        ):
+            content_item_type = FileType.METADATA
+            is_metadata = True
+
+        if content_item_type not in SKIP_RELEASE_NOTES_FOR_TYPES:
+            if content_item_type != FileType.METADATA:
+                return True, False
+            else:
+                is_metadata = True
```

Comment:
Why do we need to check if is_metadata again?

Reply:
for metadata, content_item_type can either return None or FileType.METADATA, if it's None and it is the metadata (content_item.content_type.value == Pack.content_type.value) then update the content_item_type accordingly. 

we need this in the case find_type returned FileType.METADATA instead of None.